backend/api/api.py

```diff
@@ -131,7 +129,12 @@ async def get_result(
     if not list(filter(lambda name: name == test_name, test_names)):
         raise HTTPException(status_code=404, detail="Not Found")
 
-    return _strip_last_modified(await store.get_results(user.id, test_name))
+    results = await store.get_results(user.id, test_name)
+    print(results)
+    if not results:
+        return [{}]
```

Comment:
Enforced by pydantic and I didn't want to touch the return values so playing along... (But it's silly and needs to change eventually.)